### ISSUE TITLE
Following "Getting Started" guide results in compilation errors

### ISSUE THREAD

**Issue by KidkArolis** [pedronauck/docz#718]
Installed `docz@1.0.0-rc.3`, running `docz dev` results in:

```
✖ Docz
  Compiled with some errors in 85.70ms



 ERROR  Failed to compile with 3 errors                                                                                                                                                           8:17:06 PM

 error  in ./node_modules/docz/dist/index.esm.js

Module parse failed: Unexpected token (15:38)
You may need an appropriate loader to handle this file type.
| import { MDXProvider } from '@mdx-js/tag';
|
> const BasePlayground = loadable(() => import('./chunk.esm2.js'));
| const Playground = props => typeof window !== 'undefined' ? (createElement(Suspense, { fallback: null },
|     createElement(BasePlayground, Object.assign({}, props)))) : null;

 @ ./.docz/app/root.jsx 9:0-59 16:2-15 18:19-23 25:25-31
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./.docz/app/index.jsx

 error  in ./node_modules/docz-theme-default/dist/index.esm.js

Module parse failed: Unexpected token (245:30)
You may need an appropriate loader to handle this file type.
| `;
|
> const CodeMirror = lazy(() => import('./chunk.esm2.js'));
| const getLanguage = (children) => {
|     const defaultLanguage = 'jsx';

 @ ./.docz/app/root.jsx 11:0-39 17:29-34
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./.docz/app/index.jsx

 error  in ./.docz/app/imports.js

Module parse failed: Unexpected token (8:11)
You may need an appropriate loader to handle this file type.
| export var imports = {
|   'content/index.md': function contentIndexMd() {
>     return import(
|     /* webpackPrefetch: true, webpackChunkName: "content-index" */
|     'content/index.md');

 @ ./.docz/app/root.jsx 12:0-36 26:13-20
 @ ./.docz/app/index.jsx
 @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./.docz/app/index.jsx
```

**Comment by KidkArolis** [comment]
Tried reinstalling node_modules. Didn't help. Using npm.

**Comment by davidtaboas** [comment]
@pedronauck @KidkArolis Same error reported here: https://github.com/pedronauck/docz/issues/704

The solution from https://github.com/pedronauck/docz/issues/596 doesn't work because this is another error regarding the webpack handler to load some files

**Comment by JoeCMorgan** [comment]
Any update on this?

**Comment by pedronauck** [comment]
Closed in favor of #704, the answer is there.
